src/components/Card.tsx

```diff
@@ -0,0 +1,32 @@
+import { CardType } from "../types/CardType";
+import "./Card.css";
```

Comment:
Something to think about! this is the only file that isn't using css modules and instead is just using regular css. I know this was done because there was an issue with how `classnames` sets the classes in the element, but since the other components are using CSS modules it would be better to decide which approach you want to use and not have 2 different ones. If you decide to go with regular CSS, you'd have to change the css file names, the styles import in the component and the classes in the elements from being derived from the module to just be a string. Let me know what you think!